modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,72 +3,146 @@
-
-ADD example-cr2.yaml /kustomize/plugin/ran.openshift.io/v1/policygentemplate/source-crs/
-ADD example-cr1.yaml /kustomize/plugin/ran.openshift.io/v1/policygentemplate/source-crs/
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-dev" 
+  namespace: "ztp-clusters" 
+spec:
+  bindingRules:
+    dev: "true" 
+  mcp: "master" 
+  sourceFiles:
+    # These policies/CRs come from the internal container Image
+    #Cluster Logging
+    - fileName: ClusterLogNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-ns" 
+    - fileName: ClusterLogOperGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-operator-group" 
+    - fileName: ClusterLogSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-sub" 
+    #Local Storage Operator
+    - fileName: StorageNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-ns" 
+    - fileName: StorageOperGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-operator-group" 
+    - fileName: StorageSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-sub" 
+    # Performance Addon Operator
+    - fileName: PaoSubscriptionNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-ns" 
+    - fileName: PaoSubscriptionCatalogSource.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-cat-source" 
+      spec:
+        image: <image URL here>
+    - fileName: PaoSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-sub" 
+    #These are custom local polices that come from the source-crs directory in the git repo
+    #Elasticsearch Operator
+    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-ns" 
+    - fileName: elasticsearch/ElasticsearchOperatorGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-operator-group" 
+    #Bell Custom Resources
+    - fileName: bell/apiserver-config.yaml
+      remediationAction: inform 
+      policyName: "group-dev-apiserver-config" 
+    - fileName: bell/disable-nic-lldp.yaml
+      remediationAction: inform 
+      policyName: "group-dev-disable-nic-lldp" 
 ----
+<1> Set `fileName` to include the custom CR subfolder from the `/source-crs` parent.
 
-. Open a terminal at the `ztp-update/` folder and rebuild the container:
-+
-[source,terminal,subs="attributes+"]
-----
-$ podman build -t ztp-site-generate-rhel8-custom:v{product-version}-custom-1
-----
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP Argo CD policies application.
 
-. Push the built container image to your disconnected registry, for example:
+. Update the `ClusterGroupUpgrade` CR to include the changed `PolicyGenTemplate` and save it as `cgu-test.yaml`.
```

Comment:
```suggestion
. Update the `ClusterGroupUpgrade` CR to include the changed `PolicyGenTemplate` and save it as `cgu-test.yaml`, as shown in the following example:
```
- Nit: Consider add introductory phrase to the block of sample code followed by a colon. If you want to make the change, you can add this to the Jira for fixing the alignment of the example output.

Reply:
Thanks, Changed this in new PR
https://github.com/openshift/openshift-docs/pull/64949